### PR TITLE
incubator/azuremonitor-containers - August 2019 release - (Bug fixes & Cluster Health Private preview)

### PR DESCRIPTION
**Pull request by vishiy**
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Review by rashmichandrashekar — APPROVED** [review]

**Comment by lachie83** [comment]
@vishiy @rashmichandrashekar can you please handle this before we add you to the OWNERS file.

> The following users are mentioned in OWNERS file(s) but are not members of the helm org.
> 
> Once all users have been added as members of the org, you can trigger verification by writing `/verify-owners` in a comment.
> 
> * vishiy
>   
>   * incubator/azuremonitor-containers/OWNERS
> * rashmichandrashekar
>   
>   * incubator/azuremonitor-containers/OWNERS

File: incubator/azuremonitor-containers/templates/omsagent-rbac.yaml [hunk 1]
```diff
@@ -21,6 +21,12 @@ rules:
 - apiGroups: [""]
   resources: ["pods", "events", "nodes", "namespaces", "services"]
   verbs: ["list", "get", "watch"]
+- apiGroups: ["extensions"]
```
Comment (lachie83) [inline incubator/azuremonitor-containers/templates/omsagent-rbac.yaml:24]: You might want to future proof this given this API is removed in 1.16 - https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/

File: incubator/azuremonitor-containers/templates/omsagent-crd.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```
Comment (lachie83) [inline incubator/azuremonitor-containers/templates/omsagent-crd.yaml:1]: Is this CRD used by this chart? If not, where are the docs on how to use it?

Comment (vishiy) [reply incubator/azuremonitor-containers/templates/omsagent-crd.yaml:1]: Its for internal use (health state). I will update readme to say so.

File: incubator/azuremonitor-containers/templates/omsagent-rbac.yaml (hunk 1 above)
Comment (vishiy) [reply incubator/azuremonitor-containers/templates/omsagent-rbac.yaml:24]: yes, we have whole bunch of changes coming here (to support 1.16) - in the upcoming release.We will be updating all these are part of that.

File: incubator/azuremonitor-containers/templates/omsagent-crd.yaml (hunk 2 above)
Comment (vishiy) [reply incubator/azuremonitor-containers/templates/omsagent-crd.yaml:1]: added to readme.

**Pull request merged**